config/config.go

```diff
@@ -19,6 +19,9 @@ type PanaceaConfig struct {
 	GRPCAddr         string `mapstructure:"grpc-addr"`
 	DefaultGasLimit  uint64 `mapstructure:"default-gas-limit"`
 	DefaultFeeAmount string `mapstructure:"default-fee-amount"`
+	PrimaryAddr   string `mapstructure:"primary-addr"`
+	WitnessesAddr string `mapstructure:"witnesses-addr"`
```

Comment:
This should probably be able to accept multiple values.(separated by `,`)
Because `Witnessess` can receive multiple addresses

Reply:
Ah, you have already set it to separate with whitespace. I think it would be better to separate them with a comma. :)

Reply:
Thanks, I fix it in [da46b93](https://github.com/medibloc/panacea-doracle/pull/12/commits/da46b93be929962bac9329a82cf37c1e408b84d4).